examples/gallery/maps/country_polygons.py

```diff
@@ -0,0 +1,71 @@
+"""
+Highlight country and continent polygons
+----------------------------------------
+The :meth:`pygmt.Figure.coast` method can
+highlight country polygons via the ``dcw``
+parameter. It accepts the country code, and
+can draw its borders and add a color to its landmass.
```

Comment:
Actually one can also state the whole country name.
For clarity, we maybe should add that "country code" refers to the [2-character ISO 3166-1 alpha-2 convention](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2)?

Reply:
Apparently it's possible to use the full country name (e.g. Norway instead of NO) according to https://docs.generic-mapping-tools.org/6.4/coast.html#e? But haven't tested this before.

Reply:
Yes it is possible 🙂. Here is a small example (see also my comment https://github.com/GenericMappingTools/pygmt/pull/2428#discussion_r1186381824 below):
```python
import pygmt

fig = pygmt.Figure()

fig.coast(
    region=[-12, 32, 34, 72],
    projection="M6c",
    land="gray",
    water="white",
    shorelines="1/0.1p,gray30",
    dcw=[
        "GB+gseagreen",
        "IT+p0.75p,red3",
        "ES+p0.75p,magenta4,-",
        "RO+p1p,black,.",
        "DE+gorange+p1p,dodgerblue4",
        "France+p1p,steelblue",
        "Norway,Sweden,Finland+gpink",
    ],
    frame=True,
)
fig.show()
# fig.savefig(fname="dcw_longname.png")
```

_Output figure_:
![dcw_longname](https://github.com/GenericMappingTools/pygmt/assets/94163266/fb81de7c-9781-4473-89d2-48dec5d4cb42)